### ISSUE TITLE
Error on Wayland on 4.3.0

### ISSUE THREAD

**Issue by KarolTrzeszczkowski** [Electron-Cash/Electron-Cash#2615]
Hi, I turned on the wallet on Wayland and got an error:

`Downloads/Electron-Cash-4.3.0-x86_64.AppImage                                                                                                                                    10:27
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
qt.qpa.wayland: Wayland does not support QWindow::requestActivate()
The Wayland connection broke. Did the Wayland compositor die?
`
It was on the initial wizard.

**Comment by jonas-lundqvist** [comment]
Is this a regression? Did it work pre-4.3.0?

**Comment by KarolTrzeszczkowski** [comment]
@jonas-lundqvist yes it worked and I verified on the previous binary I have it works.
